### ISSUE TITLE
Adjusting Size leads to Issues with Input Placement

### ISSUE THREAD

**Issue by Michai9011** [mlnc21/PowerApps-SVG-Merge-Modul#1]
First of all let me thank you for your work. This component has been great and a pleasure to be able to work with. I learned a lot about the capabilities of a component thanks to you.

However I am facing an issue.

I need the component to have the size:

1152x768 aka 150%x100%

If i leave the setting at 100%x100% it will work even if i make evrything bigger however now that I have a higher width than hight i get a lot of issues with my pen-input placement and I can not figure out how to resolve it.

I tried adjusting viewbox parameters and background and pensize but whatever I try its not solving my issue.

I hope to find some help here what to do to be able to use this asymetric size and still have my pen.input match, once I have done tht i think theres a lot of use cases for this.

**Comment by mlnc21** [comment]
Hi @Michai9011 ,
First, thank you 💯. 😍
Try to set the `ImageBackgroundPosition` to  `ImagePosition.Fill`.
You can see, on my Test, I don't have any issue with the size.

![image](https://user-images.githubusercontent.com/80674540/207259419-92243e72-8e75-4bb6-b40e-51045e44b635.png)

Let me know, this change solved the Issue.
Have a nice day.

**Comment by Michai9011** [comment]
I feel stupid now :D

All I had to do was change my ImageBackgroundPosition from Fit to Fill and now it works without any issue.

Bc I know as a dev its always interesting to see what people make out of smth youve build:

Im using this component as part of an inspection application where a map is being pulled and modified. I also exchanged your text component with an indexed gallery "grid" so people can select specific regions of the picture and get to a form related to them.

I have not worked with components before and found your approach very smart and easy to follow/modify for my use case, i think this one has massive potential as its very un-intuitive microsft has not yet provided a way to merge pen_input+underlying asset (which makes the whole pen input a bit less usefull)

Im very happy i found this by "accident" haha

Also in the process i got a whole new understanding of the power of base64 and how i can use it for other things too.
